exercises/phone-number/test/phone_number_test.dart

```diff
@@ -0,0 +1,66 @@
+import 'package:test/test.dart';
+import 'package:phone_number/phone_number.dart';
+
+final phoneNumber = new PhoneNumber();
+
+void main() {
+  group("PhoneNumber: Accept only numbers - ", numbersOnlyTest);
+  group("PhoneNumber: Number length - ", numberLenghtTest);
+  group("PhoneNumber: Cleanup special characters - ", cleanUpTest);
+  group("PhoneNumber: Area/Exchange Code - ", codeRangeTest);
+  group("PhoneNumber: Edge cases - ", edgeCases);
+}
```

Comment:
Did you review the edge case, should I keep it.?

Reply:
I think it is fine. The other test that I see as similar is "invalid with punctuation", but the difference with the edge case is that it would be valid if not for the special character, correct?